lib/taskana-core/src/main/java/pro/taskana/task/api/TaskService.java

```diff
@@ -119,16 +118,30 @@ Task forceCompleteTask(String taskId)
           NotAuthorizedException;
 
   /**
-   * Persists a not persisted Task which does not exist already.
+   * Persists a not persisted Task which does not exist already. <br>
+   * The default values of the created Task are:
+   *
+   * <ul>
+   *   <li><b>id</b> - generated by {@linkplain pro.taskana.common.internal.util.IdGenerator
+   *       IdGenerator}
```

Comment:
Why do you link the IdGenerator only here and not in the two lines below?

Reply:
I think we achieve a better readability if we link only the first occurrence. In the COP we could discuss the precise rules for our JavaDocs, I will propose some of them.